planner/core/expression_rewriter.go

```diff
@@ -1491,6 +1494,14 @@ func (er *expressionRewriter) funcCallToExpression(v *ast.FuncCallExpr) {
 	}
 }
 
+func (er *expressionRewriter) toTable(v *ast.TableName) {
+	val := &expression.Constant{
+		Value:   types.NewDatum(v.Name.L),
+		RetType: types.NewFieldType(mysql.TypeString),
+	}
+	er.ctxStackAppend(val, types.EmptyName)
```

Comment:
Does any place use this element in the stack?

Reply:
only nextval / lastval / setval function will take it as string args. (this function will be added in next pr)

Reply:
Should we guarantee the stack empty at the end of the rewrite stage? If we should, does this element break our guarantee?

Reply:
yes we can, take `nextval(tableName)` for example, tableName will be rewritten as a constant string, then the upper `FuncCallExpr` will consume it.